lib/bolt/cli.rb

```diff
@@ -544,7 +544,7 @@ def bundled_content
       # We only need to enumerate bundled content when running a task or plan
       content = { 'Plan' => [],
                   'Task' => [],
-                  'Plugin' => %w[puppetdb pkcs7 prompt terraform task] }
+                  'Plugin' => Bolt::Plugin::BUILTIN_PLUGINS }
```

Comment:
Do we also want to append the 'azure_inventory` module here?

Reply:
Yes, eventually we will want to move most of these to modules (notably *not* pdb or prompt).